clients/payment/src/components/SddPayment.tsx

```diff
@@ -157,6 +158,7 @@ export const SddPayment = ({ paymentLink, nonEeaCountries, setMandateUrl, large
             )
             .tapOk(mandateUrl => {
               setMandateUrl(mandateUrl);
```

Comment:
Is the `setMandateUrl(mandateUrl)` still useful as you redirect immediately after?